### PR TITLE
Feat/issue 32 ppe step by step guidance

### PR DESCRIPTION
**Pull request by ericjiang97**
Adds the PPE On Guidance Screen 😷

## Changes

- Added PPE On Guidance
- Created PPEInfo Model for (PPE Off too)
- Reworked ReusableCard into a base card and make ReusableCard implement the base card.

## Screenshots

https://ejiang.d.pr/j1OS7O

## Things to note
- Reworked ReusableCard into a base card and make ReusableCard implement the base card.

### CODE REVIEW

File: lib/widget/cards/ppe_step_card.dart [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                      softWrap: true,
+                      style: AppStyles.textP,
+                    ))
+                .toList()));
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return ReusableCardBase(
+      margin: EdgeInsets.fromLTRB(0, 10, 0, 10),
```
Comment (hades-6) [inline lib/widget/cards/ppe_step_card.dart]: Should this be 8? or 12?

File: lib/widget/cards/ppe_step_card.dart [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        Padding(
+          padding: EdgeInsets.fromLTRB(16, 0, 16, 12),
+          child: Text(
+            step.step,
+            style: AppStyles.textH3,
+          ),
+        ),
+        step.image,
+        Padding(
+          padding: EdgeInsets.fromLTRB(16, 12, 16, 5),
```
Comment (hades-6) [inline lib/widget/cards/ppe_step_card.dart]: 4 or 8? not 5?

Comment (ericjiang97) [reply lib/widget/cards/ppe_step_card.dart]: Yeah, my bad. Fixed it :)

File: lib/widget/cards/ppe_step_card.dart (hunk 1 above)
Comment (ericjiang97) [reply lib/widget/cards/ppe_step_card.dart]: Yeah, my bad. Fixed it :)

**Comment by lukesleeman** [comment]
👋 Hey @ericjiang97 I've had a look at whats going on with this PR.  It seems that flutter analyzer is failing with the following:

```

  error • The method 'PPEOnGuidance' isn't defined for the type 'MyApp' • lib/main.dart:36:44 • undefined_method
  error • The return type 'dynamic' isn't a 'Widget', as required by the closure's context • lib/main.dart:36:44 • return_of_invalid_type_from_closure
   info • Prefer relative imports for files in `lib/` • lib/view/ppe/ppe_on_guidance.dart:4:8 • prefer_relative_imports
   info • Prefer relative imports for files in `lib/` • lib/view/ppe/ppe_on_guidance.dart:5:8 • prefer_relative_imports
   info • Prefer relative imports for files in `lib/` • lib/view/ppe/ppe_on_guidance.dart:6:8 • prefer_relative_imports
   info • Prefer relative imports for files in `lib/` • lib/widget/cards/ppe_step_card.dart:4:8 • prefer_relative_imports
   info • Prefer relative imports for files in `lib/` • lib/widget/cards/ppe_step_card.dart:5:8 • prefer_relative_imports
   info • Prefer relative imports for files in `lib/` • lib/widget/cards/ppe_step_card.dart:6:8 • prefer_relative_imports
   info • Prefer declaring const constructors on `@immutable` classes • lib/widget/cards/ppe_step_card.dart:12:3 • prefer_const_constructors_in_immutables
   info • Prefer const with constant constructors • lib/widget/cards/ppe_step_card.dart:16:18 • prefer_const_constructors
   info • Prefer const with constant constructors • lib/widget/cards/ppe_step_card.dart:31:15 • prefer_const_constructors
   info • Prefer const with constant constructors • lib/widget/cards/ppe_step_card.dart:33:16 • prefer_const_constructors
   info • Prefer const with constant constructors • lib/widget/cards/ppe_step_card.dart:37:20 • prefer_const_constructors
   info • Prefer const with constant constructors • lib/widget/cards/ppe_step_card.dart:45:20 • prefer_const_constructors
   info • Prefer const with constant constructors • lib/widget/cards/ppe_step_card.dart:49:20 • prefer_const_constructors
   info • Prefer relative imports for files in `lib/` • lib/widget/cards/reusable_card_base.dart:2:8 • prefer_relative_imports
   info • Unused import: 'package:wh_covid19/style.dart' • lib/widget/cards/reusable_card_base.dart:2:8 • unused_import
   info • Prefer asserts with message • lib/widget/cards/reusable_card_base.dart:49:8 • prefer_asserts_with_message
   info • Place 'package:' imports before relative imports • lib/widget/reusable_card.dart:3:1 • directives_ordering
   info • Prefer relative imports for files in `lib/` • lib/widget/reusable_card.dart:3:8 • prefer_relative_imports
   info • Prefer if elements to conditional expressions where possible • lib/widget/reusable_card.dart:53:9 • prefer_if_elements_to_conditional_expressions
```

Here is a link: https://codemagic.io/app/5e75f4bfdd1d6d000a198e90/build/5e7c8fc929657a000c2e45e5

I don't know why that's not getting reported up to the build status here, and its just showing "Some checks haven't completed"

You can check on the build statuses via the `#covid-19-app-builds` channel in our slack.  Thats where I saw these errors.

**Review by domesticmouse — APPROVED** [review]

**Pull request merged**